lib/geckoboard/dataset.rb

```diff
@@ -16,6 +16,10 @@ def put(data)
       client.put_data(id, payload_formatter.format(data))
     end
 
+    def post(data, options={})
```

Comment:
It may also be cool / idiomatic to implement `#<<`.

```ruby
dataset = client.datasets.find_or_create(...)
dataset << { amount: 100 }
```

What do you think?